### ISSUE TITLE
hotkey for turning a each line in a selection to list items.

### ISSUE THREAD

**Issue by rkennesson** [vnotex/vnote#251]
v1.16, Kubuntu 18.04

Is there a hotkey to turn a selection of text to turn each line into a list item?

**Comment by tamlok** [comment]
No, there isn't any for now. I am figuring out a way to let snippet to call a user-provided script to manipulate the text. After this, user could do whatever they want.

Thanks!

**Comment by rkennesson** [comment]
Is it possible to leverage the code that makes the TAB key insert a tab before each line (to make a code block) and instead add `* ` before each line?

I don't know what part of the code controls the TAB key making a code block so maybe it wouldn't work that way.

**Comment by tamlok** [comment]
Supported.
